app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -470,7 +494,7 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
                                 startActivity(browserIntent)
                                 return true
                             } else {
-                                Log.d(TAG, "No unique cases found to override")
+                                // Log.d(TAG, "No unique cases found to override url=${it}") // Unstoppable flood.
```

Comment:
Please revert. We don't leave commented code like this, and if you're seeing this statement a lot that is probably an issue with your setup (it is only called for main page navigation/redirects, so with a normal setup you don't see it unless trying to navigate to outside). Some of the log statements you added will generate more lines than this one.

Reply:
Without a URL, the information content of this log is poor.
This log shows the correct URL, which has no additional action.

For example, 3 logs per second:  https://community.home-assistant.io/t/apps-cant-access-ha/534210/3

I was receiving more than 10 logs per second.

Reply:
Adding an url could be useful but you're commenting it out so in the end adding no value. And as mentioned, we don't leave in commented code like this.

> I was receiving more than 10 logs per second.

Did you manage to find out what caused it? If so, that could be discussed in an issue and/or changed/fixed in a different PR.

Reply:
The reason is that shouldOverrideUrlLoading() is constantly being called on another thread.
shouldOverrideUrlLoading() is used to filter special URLs.
If the URL is normal, then the flood will be in the log.

It's better to delete this line.